src/static_array.cr

```diff
@@ -65,7 +65,7 @@ struct StaticArray(T, N)
   def self.new(& : Int32 -> T)
     array = uninitialized self
     buf = array.to_unsafe
-    # Using a macro prevents slow compile times: https://github.com/crystal-lang/crystal/issues/2485
+    # Unroll the loop in Crystal land to avoid slow compile times in release mode. See https://github.com/crystal-lang/crystal/issues/2485#issuecomment-644416515
     {% for i in 0...N %}
     buf[{{i.id}}] = yield {{i.id}}
```

Comment:
```suggestion
      buf[{{i.id}}] = yield {{i.id}}
```

Reply:
Just out of curiosity, is there a standard in Crystal for indenting macro bodies versus aligning them with the surrounding code where they'll be generated? For me, I think aligning it where it will be generated is more intuitive, not that it really matters though 🤷🏼‍♀️

Reply:
The most _standard_ approach is the one used by the builtin formatter - which indents them (for simple bodies at least). I do the same since `{% ... %}` macro blocks define a scope.

Reply:
Also the standard way is to write `{{i}}` instead of `{{ i }}` - don't know why